core/templates/components/summary-tile/story-summary-tile.component.html

```diff
@@ -1,119 +1,116 @@
 <div class="story-summary-tile protractor-test-story-summary-title">
   <div class="story-progress-bar">
-    <svg class="story-progress-svg" width="50" height="50" ng-if="$ctrl.nodeCount">
+    <svg class="story-progress-svg" width="50" height="50" *ngIf="nodeCount">
       <circle class="story-progress-circle total-circle"
               cx="25"
               cy="25"
               r="20"
-              stroke-dasharray="<[$ctrl.getStrokeDashArrayValues()]>"
+              [attr.stroke-dasharray]="getStrokeDashArrayValues()"
               stroke-dashoffset="-2.5"
               stroke-linecap="round"/>
       <circle class="story-progress-circle completed-circle"
               cx="25"
               cy="25"
               r="20"
-              stroke-dasharray="<[$ctrl.getCompletedStrokeDashArrayValues()]>"
+              [attr.stroke-dasharray]="getCompletedStrokeDashArrayValues()"
               stroke-dashoffset="-2.5"
-              stroke-linecap="<[$ctrl.completedStoriesCount ? 'round' : 'butt']>"/>
+              [attr.stroke-linecap]="completedStoriesCount ? 'round' : 'butt'"/>
       <text x="25"
             y="-20"
             text-anchor="middle"
             stroke="#000"
             stroke-width="0.5"
             font-size="0.9em"
             transform="rotate(90)">
-        <[$ctrl.storyProgress]>%
+        {{ storyProgress }}%
       </text>
     </svg>
   </div>
   <div class="oppia-story-title">
     <strong class="title-text">
-      <a ng-href="<[$ctrl.getStoryLink()]>" ng-class="{'disabled-link': !$ctrl.classroomUrlFragment}"><[$ctrl.storySummary.getTitle()]> </a>
+      <a [href]="getStoryLink()" [ngClass]="{'disabled-link': !classroomUrlFragment}">{{ storySummary.getTitle() }} </a>
```

Comment:
Can you precompute `getStoryLink()` and `storySummary.getTitle()` in ngOnInit and use those values instead.